src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -212,6 +221,44 @@ protected FieldBuilder(FieldType fieldType)
         public virtual FieldBuilder<TSourceType, TReturnType> Argument<TArgumentGraphType>(string name, Action<QueryArgument>? configure = null)
             where TArgumentGraphType : IGraphType => Argument(typeof(TArgumentGraphType), name, configure);
 
+        /// <summary>
+        /// Adds an argument to the field.
+        /// </summary>
+        /// <typeparam name="TArgumentClrType">The clr type of the argument.</typeparam>
+        /// <param name="name">The name of the argument.</param>
+        /// <param name="nullable">Indicates if the argument is optional or not.</param>
+        /// <param name="configure">A delegate to further configure the argument.</param>
+        public virtual FieldBuilder<TSourceType, TReturnType> Argument<TArgumentClrType>(string name, bool nullable = false, Action<QueryArgument>? configure = null)
+        {
+            Type type;
+
+            try
+            {
+                type = typeof(TArgumentClrType).GetGraphTypeFromType(nullable, TypeMappingMode.InputType);
+            }
+            catch (ArgumentOutOfRangeException exp)
+            {
+                throw new ArgumentException($"The GraphQL type for argument '{FieldType.Name}.{name}' could not be derived implicitly from type '{typeof(TArgumentClrType).Name}'.", exp);
+            }
+
+            return Argument(type, name, configure);
+        }
+
+        /// <summary>
+        /// Adds an argument to the field.
+        /// </summary>
+        /// <typeparam name="TArgumentClrType">The clr type of the argument.</typeparam>
+        /// <param name="name">The name of the argument.</param>
+        /// <param name="nullable">Indicates if the argument is optional or not.</param>
+        /// <param name="description">The description of the argument.</param>
+        /// <param name="configure">A delegate to further configure the argument.</param>
+        public virtual FieldBuilder<TSourceType, TReturnType> Argument<TArgumentClrType>(string name, bool nullable, string? description, Action<QueryArgument>? configure = null)
```

Comment:
See above:
```
[Obsolete("Please use Action<QueryArgument> parameter from other Argument() method overloads to set default value for parameter or use Arguments() method. This method will be removed in v8.")]
        public virtual FieldBuilder<TSourceType, TReturnType> Argument<TArgumentGraphType, TArgumentType>(string name, string? description,
            TArgumentType? defaultValue = default, Action<QueryArgument>? configure = null)
```

I would like to minimize API surface here too as well as I did for `FieldXXX` methods.

Reply:
I matched what you had done previously -- that is why there is no overload with a `defaultValue` param. Lmk if there's anything else you'd like done.

Reply:
Ah, I see existing overload with description. I agree that `description` property is more widely used (I'm sure) than `defaultValue`. I'm fine to keep it as is for now.